File: 백준/Silver/2563. 색종이/색종이.py
#boj 2563
import sys

ary = [[0 for _ in range(101)] for _ in range(101)]
result = 0
N = int(input())

for _ in range(N):
    x,y = list(map(int,sys.stdin.readline().split()))

    for i in range(x,x+10):
        for j in range(y,y+10):
            ary[i][j] = 1

for i in ary:
    result +=sum(i)

print(result)

File: 백준/Silver/2563. 색종이/색종이.js
// 백준 : 색종이
const [N,...input] = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(x => x.split(' ').map(Number));

let arr = Array.from({length : 100},() => Array(100).fill(0));
let size = 0 ;
let tmp = 0;

for(let i = 0 ; i < N ; i++){
    let [a,b] = input[i];
    for(let row = a ; row < a+10; row++){
        for(let col = b ; col < b+10 ; col++){
            if(arr[row][col] === 1) continue;
            arr[row][col]+=1;
            size++;
        }
    }
}

console.log(size);
